### DISCUSSION TITLE
Re-start polling on change in script

### DISCUSSION THREAD

**Discussion by tech-savvy-guy** [eternnoir/pyTelegramBotAPI#1634] [Ideas]
Hi, I was working with a Flask application and noticed an interesting feature.

```python
import flask

app = flask.Flask(__name__)

if __name__ == '__main__':
    app.run(debug=True)

```

Whenever there is a change in the script, the library detects it and re-starts automatically.

![image](https://user-images.githubusercontent.com/83786816/179349395-736e5da4-ded8-4ea0-8409-e6dafd9ad550.png)

I was thinking maybe something similar can be implemented with the `polling` method?

**Comment by coder2020official** [comment]
Good idea

**Comment by tech-savvy-guy** [comment]
So when can we expect this feature?

**Comment by coder2020official** [comment]
First of all, I should find a way to implement it, then discuss with collaborator of the library, and then find time to implement it.

**Comment by tech-savvy-guy** [comment]
Can you just discuss this with the collaborator? Whether this is a feature that he would actually want in the library. Then maybe I can research on how to implement this?

**Comment by coder2020official** [comment]
I have a lot of things in the backlog, and there are some important ones. Let me complete my checklist first.

**Comment by tech-savvy-guy** [comment]
No, I mean I want to contribute. So maybe I can work on this feature. I just need the approval from the collaborator.

**Comment by coder2020official** [comment]
#1693
